mapswipe_workers/mapswipe_workers/utils/user_management.py

```diff
@@ -1,167 +1,157 @@
+        ref = fb_db.reference(f"v2/users/{user.uid}/")
+        ref.update(
+            {
+                "username": username,
+                "taskContributionCount": 0,
+                "groupContributionCount": 0,
+                "projectContributionCount": 0,
+                "created": datetime.datetime.utcnow().isoformat()[0:-3]
+                + "Z",  # Store current datetime in milliseconds
+            }
         )
-
-        ref = fb_db.reference(f'v2/users/{user.uid}/')
-        ref.update({
-            'username': username,
-            'taskContributionCount': 0,
-            'groupContributionCount': 0,
-            'projectContributionCount': 0,
-            'created': datetime.datetime.utcnow().isoformat()[0:-3]+'Z' # Store current datetime in milliseconds
-        })
-        logger.info(f'created new user: {user.uid}')
+        logger.info(f"created new user: {user.uid}")
         return user
     except Exception as e:
-        logger.info(f'could not create new user {email}.')
+        logger.info(f"could not create new user {email}.")
         raise CustomError(e)
 
 
 def delete_user(email):
     fb_db = firebaseDB()
     try:
         user = auth.get_user_by_email(email)
-        ref = fb_db.reference(f'v2/users/')
-        ref.update({
-            user.uid: None
-        })
+        ref = fb_db.reference(f"v2/users/")
+        ref.update({user.uid: None})
         auth.delete_user(user.uid)
-        logger.info(f'deleted user {email}')
+        logger.info(f"deleted user {email}")
     except Exception as e:
-        logger.info(f'could not find user {email} in firebase to delete.')
+        logger.info(f"could not find user {email} in firebase to delete.")
         raise CustomError(e)
 
 
 def sign_in_with_email_and_password(email, password):
-    api_key = CONFIG['firebase']['api_key']
-    request_ref = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key={0}".format(api_key)
+    api_key = FIREBASE_API_KEY
+    request_ref = "https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key={0}".format(
+        api_key
+    )
     headers = {"content-type": "application/json; charset=UTF-8"}
     data = json.dumps({"email": email, "password": password, "returnSecureToken": True})
     request_object = requests.post(request_ref, headers=headers, data=data)
     current_user = request_object.json()
-    logger.info(f'signed in with user {email}')
+    logger.info(f"signed in with user {email}")
     return current_user
 
 
 def get_firebase_db(path, custom_arguments=None, token=None):
-    databaseName = CONFIG['firebase']['database_name']
-    database_url = f'https://{databaseName}.firebaseio.com'
-    request_ref = '{0}{1}.json?{3}auth={2}'.format(database_url, path, token, custom_arguments)
-    headers = {"content-type": "application/json; charset=UTF-8"}
-    request_object = requests.get(
-        request_ref,
-        headers=headers
+    databaseName = FIREBASE_DB
```

Comment:
maybe picky, but why `databaseName` instead of `database_name`

Reply:
No reason. I changed it.